### DISCUSSION TITLE
Clarification on License for commercial use for Exhaustive Defintions

### DISCUSSION THREAD

**Discussion by mouldao** [MediatedCommunications/Mime-Detective#21] [Q&A]
Would like to confirm that for commercial use for the exhaustive definition pack, a license needs to be paid at https://mark0.net/ for TrID - File Identifier?

**Comment by TonyValenti** [comment]
Yes. That is correct.

**Comment by atimofte29** [comment]
Were you able to identify a commercial subscription on this site https://mark0.net/? I could only find a donation link. 
I did not see anywhere a price (like the 300 €) for a subscription, like indicated in the documentation.
Furthermore, the package itself has an MIT license, so it is a bit confusing ...

**Comment by slm-2015** [comment]
Hello - I'd like a commercial license as well, but cannot find anywhere on https://mark0.net/ to pay to acquire one. Any pointers please? Thanks.

**Reply by TonyValenti** [reply]
You have to contact him via email.

**Reply by slm-2015** [reply]
Thanks Tony. I've emailed him on his gmail address.
